tests/topotests/static_routing_with_ibgp/test_static_routes_topo2_ibgp.py

```diff
@@ -0,0 +1,2357 @@
+
+            step(
+                "After removing the static route with N1 to N8 one by one , "
+                "route become active with next preferred nexthop and nexthop "
+                "which got removed is not shown in RIB and FIB"
+            )
+            nh = NEXT_HOP_IP["nh" + str(nhp)][addr_type]
+            result = verify_rib(
+                tgen,
+                addr_type,
+                dut,
+                input_dict_4,
+                next_hop=nh,
+                protocol=protocol,
+                expected=False,
+            )
+            assert result is not True, "Testcase {} : Failed \nError: Route "
+            " is still present in RIB".format(tc_name)
+
+        step("Reconfigure the deleted routes and verify they are installed")
+        for nhp in range(1, 9):
+            input_dict_4 = {
+                "r2": {
+                    "static_routes": [
+                        {
+                            "network": PREFIX1[addr_type],
+                            "next_hop": NEXT_HOP_IP["nh" + str(nhp)][addr_type],
+                            "admin_distance": 10 * nhp,
+                        }
+                    ]
+                }
+            }
+            logger.info("Configure static routes")
+            result = create_static_routes(tgen, input_dict_4)
+            assert result is True, "Testcase {} : Failed \n Error: {}".format(
+                tc_name, result
+            )
+
+            dut = "r2"
+            protocol = "static"
+            nh = NEXT_HOP_IP["nh1"][addr_type]
+            result = verify_rib(tgen, addr_type, dut, input_dict_4, protocol=protocol)
+            assert result is True, "Testcase {} : Failed \nError: Route "
+            " is still present in RIB".format(tc_name)
+
+            step("Reload the FRR router")
+            # stop/start -> restart FRR router and verify
+            stop_router(tgen, "r2")
+            sleep(5)
+            start_router(tgen, "r2")
+
+            step("After reloading, verify that routes are still present in R2.")
+            result = verify_rib(
+                tgen,
+                addr_type,
+                dut,
+                second_rte,
+                next_hop=nh,
+                protocol=protocol,
+                fib=True,
+            )
+            assert result is True, (
+                "Testcase {} : Failed \nError: Route "
+                " is missing in RIB".format(tc_name)
+            )
+
+        dut = "r2"
+        for intfr in range(0, 8):
+            intf = topo["routers"]["r2"]["links"]["r3-link{}".format(intfr)][
+                "interface"
+            ]
+            shutdown_bringup_interface(tgen, dut, intf, False)
+
+        dut = "r3"
+        for intfr in range(0, 8):
+            intf = topo["routers"]["r3"]["links"]["r2-link{}".format(intfr)][
+                "interface"
+            ]
+            shutdown_bringup_interface(tgen, dut, intf, False)
+        sleep(180)
```

Comment:
here too

Reply:
Changes made are:

This sleep is as per the test case , sleep timer is optimised to 6 secs.
Base config is modified to use bgp aggressive timers.